internal/ct/handlers.go

```diff
@@ -351,13 +351,13 @@ func addChainInternal(ctx context.Context, opts *HandlerOptions, log *log, w htt
 	index, dedupTimeMillis, err := log.storage.Add(ctx, entry)
 	if err != nil {
 		if errors.Is(err, tessera.ErrPushback) {
+			tooManyRequestsAttribute := fmt.Sprintf("tessera_pushback_%s", strings.Split(err.Error(), " ")[0])
```

Comment:
Also, it feels a bit cheeky to be adding a `tessera_` scoped attribute here :) (maybe that's fine, though?)